src/wp-content/themes/twentytwentythree/readme.txt

```diff
@@ -0,0 +1,31 @@
+=== Twenty Twenty-Three ===
+Contributors: wordpressdotorg
+Requires at least: 5.9
+Tested up to: 6.1
+Requires PHP: 5.7
+License: GPLv2 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+
+== Description ==
+
+
+
+== Changelog ==
+
+= 0.0.1 =
+* Initial release
```

Comment:
Let's change this to match the other default themes. When the theme goes live, this article will need to be published and can be updated as needed in case a change is forgotten here.

```suggestion
* Released: November 1, 2022

https://wordpress.org/support/article/twenty-twenty-three-changelog#Version_1.0
```